pages/linux/pkginfo.md

```diff
@@ -0,0 +1,19 @@
+# pkginfo
+
+> Query the package database on a CRUX system
+
+- list installed packages and their versions
+
+`pkginfo -i`
+
+- list files owned by a package
+
+`pkginfo -l {{package-name}}`
+
+- list the owner(s) of a file matching a pattern
```

Comment:
What happens when multiple files are matched? You get all authors lumped together?